standalone-examples/src/main/java/io/pravega/example/streamcuts/StreamCutsCli.java

```diff
@@ -0,0 +1,317 @@
+package io.pravega.example.streamcuts;
+
+import io.pravega.client.stream.ReaderGroupConfig;
+import io.pravega.client.stream.Stream;
+import io.pravega.client.stream.StreamCut;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URI;
+import java.util.AbstractMap.SimpleEntry;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+
+public class StreamCutsCli {
```

Comment:
We need test cases for this CLI. Otherwise, how do we know that a change in Pravega actually broke it?

Reply:
You are right, there are no tests for samples. As this is a separate task, I will create a separate issue focused on developing test classes for all standalone examples.